content/docs/06-integrations/00-cilium-tetragon.md

```diff
@@ -0,0 +1,41 @@
+---
+title: 'Cilium-Tetragon'
+metaTitle: 'cilium-Tetragon'
+metaDescription: 'Cilium Tetragon monitoring pack for Spectro Cloud Palette'
+hiddenFromNav: true
+isIntegration: true
+category: ['monitoring']
+logoUrl: 'https://soak.stage.spectrocloud.com/assets/monitoring_layer.3b14cf5b.svg'
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+# Cilium Tetragon
+
+Tetrogon is an eBPF based security observability and runtime enforcement. eBPF is used to safely and efficiently extend the kernel's capabilities without requiring changing kernel source code or load kernel modules. Tetragon is a Cilium community open-source project that enables profound visibility with filtering and aggregation with the eBPF collector support to deliver visibility at depth with minimal overhead.
+
+Palette supports the Cilium Tetragon as an add-on pack for monitoring services. Refer [create cluster profile](/cluster-profiles/task-define-profile#overview) page for the information on adding an add-on pack.

```

Comment:
```suggestion
Palette supports the Cilium Tetragon as an add-on pack for monitoring services. Refer to the [create cluster profile](/cluster-profiles/task-define-profile#overview) page for more information on how to use an add-on pack.
```
Language cleanup

Reply:
Remove "the". 

Palette supports Cilium Tetragon as an add-on pack for monitoring services. Refer to the [create cluster profile](/cluster-profiles/task-define-profile#overview) page for more information on how to use an add-on pack.